api/productions.js

```diff
@@ -96,8 +96,7 @@ export const getSuggestedEvents = (
     return [];
```

Comment:
We should probably also update this to return an object now, right?

Reply:
True